tests/integration_tests/test_sklearn.py

```diff
@@ -9,6 +9,18 @@
 import numpy as np
 
 
+def test__is_arraylike():
+    assert integration.sklearn._is_arraylike([])
```

Comment:
Please check numpy ndarray because the current test cases does not take care of `shape` or `__array__`.
The following is an example of the fix:
```suggestion
    assert integration.sklearn._is_arraylike([])
    assert integration.sklearn._is_arraylike(np.zeros(5))
```